### PR TITLE
Update common.libsonnet to 2.5.0

### PR DESCRIPTION
**Pull request by ManfeiBai**
# Description

update r2.5 branch test with 2.5.0 final wheel

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Comment by ManfeiBai** [comment]
cc @tengyifei

File: dags/legacy_test/tests/pytorch/r2.5/common.libsonnet [hunk 1]
```diff
@@ -147,10 +147,10 @@ local volumes = import 'templates/volumes.libsonnet';
         pip uninstall -y torch torchvision
         pip install torch==2.5.0 --index-url https://download.pytorch.org/whl/test/cpu
         pip install --user --no-use-pep517 "git+https://github.com/pytorch/vision.git@d23a6e1664d20707c11781299611436e1f0c104f"
-        pip install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl
+        pip install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0%2Blibtpu-cp310-cp310-manylinux_2_28_x86_64.whl
```
Comment (zpcore) [inline dags/legacy_test/tests/pytorch/r2.5/common.libsonnet:150]: Is ` libtpu` necessary? I tried `https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` and it also exists.

Comment (ManfeiBai) [reply dags/legacy_test/tests/pytorch/r2.5/common.libsonnet]: thanks, updated to `https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl`

cc @lsy323

File: dags/pytorch_xla/configs/pytorchxla_torchbench_config.py [hunk 2]
```diff
@@ -85,16 +85,16 @@ class R2_4(enum.Enum):
     TORCH_XLA_REPO_BRANCH = "-b v2.4.0-rc8"
 
   class R2_5(enum.Enum):
-    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
-    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0%2Blibtpu-cp310-cp310-manylinux_2_28_x86_64.whl"
```
Comment (zpcore) [inline dags/pytorch_xla/configs/pytorchxla_torchbench_config.py:88]: Also update this ` libtpu`. Then we can launch the final test :>

Comment (ManfeiBai) [reply dags/pytorch_xla/configs/pytorchxla_torchbench_config.py]: Thanks, nice catch, updated

**Review by zpcore — APPROVED** [review]
LGTM!

**Pull request merged**